mailjet_client.go

```diff
@@ -209,6 +209,20 @@ func (c *Client) Delete(mr *Request) (err error) {
 	return err
 }
 
+// Special case of Delete for API v4 (e.g. delete contacts in case of GDPR requests)
+func (c *Client) DeleteV4(mr *Request) (err error) {
```

Comment:
Based on https://dev.mailjet.com/email/guides/contact-management/#overview it looks like the v3 delete is not a thing anymore. So better just to update the origin `Delete` func.